code/API_definitions/location.yaml

```diff
@@ -181,6 +184,12 @@ components:
       minimum: 2
       maximum: 200
       example: 50
+    MaxLocationAge:
+      description: Max location aging validity required by the application in Minute
+      type: number
+      minimum: 1
+      maximum: 1440
+      example: 30
```

Comment:
default: 0

Reply:
I have to check the technical implications, but I assume that maxAge=0 implies that implementation must not rely on any kind of stale or cached data. If we set the default to this, we are forcing that requests which not include explicitly this parameter would get an error, if some implementation is caching some data for few minutes, which could not be the intention of the consumer.